etc/kubernetes/helm/enterprise-gateway/templates/daemonset.yaml

```diff
@@ -29,6 +29,9 @@ spec:
           value: !!str {{ .Values.kip.interval }}
         - name: KIP_PULL_POLICY
           value: {{ .Values.kip.pullPolicy }}
+        # Authorization token required for all requests (the token should be the same as EG_AUTH_TOKEN)
+        - name: KIP_AUTH_TOKEN   
+          value: {{ .Values.authToken }}
```

Comment:
Please update the comment.  I don't know what a non-value produces for envs, but I think this name/value pair should be conditionally introduced via `{{- if .Values.authToken }}` and `{{- end }}` wrappings.